R/wflow_toc.R

```diff
@@ -0,0 +1,40 @@
+#' Add a table of content on the clipboard
+#'
+#' \code{wfow_toc} add a table of content on the clipboard. The users can paste it
+#' in the \code{index.Rmd} document. The table of content includes all \code{html} document
+#' in the \code{docs} directory the users want to display.
+#'
+#' @param gh_url character. The Github Pages root directory.
+#' @param proj_url character. The project name.
+#'
+#' @return text character. The list of table of content in the markdown syntax.
+#' @import dplyr
+#' @import clipr
+#' @import rmarkdown
+#' @export
+
+wflow_toc <- function(project = ".") {
+  s <- wflow_status(project = ".")
+  df<- s$status
+  df$rmd_path <- rownames(df)
+  df <- df[df$published]
+  df$html_path <- to_html(df$rmd_path,outdir = s$docs) %>% basename()
+  file_name <- sapply(df$rmd_path, function(x) rmarkdown::yaml_front_matter(x)$title) %>%
```

Comment:
This part is tricky since an Rmd file may not have a title defined. I think it may be best handled with a custom function. Can I send push a commit to your pull request to add this? Then you would get the change locally by running `git pull`.

Reply:
Sure, that is great, I am looking forward to it.

Reply:
I try a pull request from your `dev` branch to mine and now I know how it works.